src/Sulu/Component/Webspace/Document/Initializer/WebspaceInitializer.php

```diff
@@ -111,12 +107,22 @@ private function initializeWebspace(OutputInterface $output, Webspace $webspace)
 
             $output->writeln(sprintf('  [+] <info>homepage</info>: [%s] %s (%s)', $homeType, $homePath, $webspaceLocale));
 
-            $this->documentManager->persist($homeDocument, $webspaceLocale, [
-                'path' => $homePath,
-                'auto_create' => true,
-                'ignore_required' => true,
-            ]);
+            $persistOptions = ['ignore_required' => true];
+            if (!$homeDocument) {
+                $homeDocument = new HomeDocument();
+                $persistOptions['path'] = $homePath;
+                $persistOptions['auto_create'] = true;
+            } else {
+                $homeDocument = $this->documentManager->find($homePath, $webspaceLocale, [
+                    'load_ghost_content' => false,
+                ]);
+            }
+
+            $homeDocument->setTitle('Homepage');
+            $homeDocument->setStructureType($homeType);
+            $homeDocument->setWorkflowStage(WorkflowStage::PUBLISHED);
```

Comment:
I think this line doesn't make sense, because the `WorkflowStage` should be correctly set by the `persist` and `publish` call afterwards. Therefore I would remove it.